projects/hslayers/src/components/save-map/form/form.component.ts

```diff
@@ -128,16 +128,13 @@ export class HsSaveMapAdvancedFormComponent implements OnDestroy, OnInit {
   /**
    * Check if user is allowed to save the composition, based on the currently selected endpoint type
    * @returns True if endpoint type is 'layman'.
-   * True if the endpoint type is 'statusmanager' and the user is authorized,
-   * false otherwise
+   * True if the user is authorized
```

Comment:
```suggestion
```

Actually the method only checks for "laymanity" of the current endpoint and for authorization